tgbotapi.core/src/commonMain/kotlin/dev/inmo/tgbotapi/requests/send/SendVenue.kt

```diff
@@ -25,7 +25,9 @@ data class SendVenue(
     @SerialName(addressField)
     val address: String,
     @SerialName(foursquareIdField)
-    val foursquareId: String? = null,
+    val foursquareId: FoursquareId? = null,
+    @SerialName(googlePlaceIdField)
+    val googlePlaceId: GooglePlaceId? = null,
```

Comment:
Here must be `googlePlaceType` too (https://core.telegram.org/bots/api#sendvenue)

Reply:
Why it's `googlePlaceType`, but `foursquareId`?

Reply:
![Screenshot_20201106-020518_Vivaldi-01](https://user-images.githubusercontent.com/6850345/98290526-fd1c8f80-1fd3-11eb-9501-ebbaa5f14a55.jpeg)
Yes, you are right, according to docs we must have both fields `googlePlaceType` and `fourSquareType` in this request

Reply:
Should I also add Foursquare type?

Reply:
I think so